### PR TITLE
Block /profiles in robots.txt and then change crawl priority.

### PR DESCRIPTION
**Pull request by burtonator**
## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/8690

## Description of Changes
- block /profile
- set crawl priority properly

## Test Plan
- write a new sitemap
- load the sitemap
- verify there are no profile URLs
- verify thread priority
- verify robots.txt has the disallow rule for /profile

## Deployment Plan

1. 

## Other Considerations

-

### CODE REVIEW

File: libs/sitemaps/src/createDatabasePaginator.ts [hunk 1]
```diff
@@ -2,10 +2,21 @@ import { models, ThreadInstance, UserAttributes } from '@hicommonwealth/model';
 import { getThreadUrl } from '@hicommonwealth/shared';
 import { Op } from 'sequelize';
 
+const THREAD_PRIORITY: number = parseFloat(
+  process.env.SITEMAP_THREAD_PRIORITY || '0.8',
+);
+const PROFILE_PRIORITY: number = parseFloat(
+  process.env.SITEMAP_PROFILE_PRIORITY || '0.4',
```
Comment (timolegros) [inline libs/sitemaps/src/createDatabasePaginator.ts:9]: Should create `libs/sitemaps/src/config.ts` and set these variables in there using the same pattern as other config files.

Comment (Rotorsoft) [reply libs/sitemaps/src/createDatabasePaginator.ts:9]: +1

Comment (burtonator) [reply libs/sitemaps/src/createDatabasePaginator.ts:9]: ok.. fair enough.

**Comment by jnaviask** [comment]
@burtonator status on this PR?

**Comment by burtonator** [comment]
I changed this so if the sitemap priority is -1 then we disable sitemaps for that profile.

**Review by timolegros — APPROVED** [review]

**Review by kurtisassad — APPROVED** [review]

**Pull request merged**